ext/opentelemetry-ext-stackdriver/tests/test_stackdriver_exporter.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2017, OpenCensus Authors
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from unittest import mock
+
+import opentelemetry.ext.stackdriver as sd_exporter
+from opentelemetry.sdk.trace import Span
+from opentelemetry.trace import SpanContext, SpanKind
+from opentelemetry.util.version import __version__
+
+
+class _Client(object):
+    def __init__(self, project=None):
+        if project is None:
+            project = "PROJECT"
+
+        self.project = project
+
+    def batch_write_spans(self, name, spans):
+        pass
+
+
+class TestStackdriverSpanExporter(unittest.TestCase):
+    def test_constructor_default(self):
+        patch = mock.patch("opentelemetry.ext.stackdriver.Client", new=_Client)
+
+        with patch:
+            exporter = sd_exporter.StackdriverSpanExporter()
+
+        project_id = "PROJECT"
+        self.assertEqual(exporter.project_id, project_id)
```

Comment:
I know it's copied from the OC tests, but I don't see the point of `_Client` here. Since we probably want to client patched for every test, you could do this instead:

```python
    def setUp(self):
        self.client_patcher = mock.patch(
            "opentelemetry.ext.stackdriver.Client"
        )
        self.client_patcher.start()

    def tearDown(self):
        self.client_patcher.stop()

    def test_constructor_default(self):
        exporter = sd_exporter.StackdriverSpanExporter()
        self.assertEqual(exporter.project_id, exporter.client.project)
```